utils/kafka-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -27,8 +27,9 @@ case class KafkaConfig(
     schemaRegistryCacheConfig: SchemaRegistryCacheConfig = SchemaRegistryCacheConfig(),
     avroAsJsonSerialization: Option[Boolean] = None,
     kafkaAddress: Option[String] = None,
-    // TODO: remove this feature flag in future release
-    useNamingStrategyForConsumerGroupId: Boolean = true
+    // TODO question: move it into a generic map / move it into nested config class / leave it here?
+    enableIdleTimeout: Boolean = true,
+    idleTimeout: FiniteDuration = 3 minutes
```

Comment:
I did it like that at first, but I want to have:
- a default timeout that is turned on if someone leaves it empty
- a way to disable the idleness entirely

Is it possible to do that with one optional field? For `outOfOrderness` it may work ok because I guess you can set the outOfOrdeness to 0 to effectively disable that, but here I think we should have a way to disable setting idleness with a default.